config/config.go

```diff
@@ -136,6 +136,7 @@ type performance struct {
 	ShardCronFrequency     time.Duration `config:"shard_cron_frequency" default:"1s"`
 	MultiplexerPollTimeout time.Duration `config:"multiplexer_poll_timeout" default:"100ms"`
 	MaxClients             int32         `config:"max_clients" default:"20000" validate:"min=0"`
+	MaxCmdHandlers         int32         `config:"max_cmd_handlers" default:"20000" validate:"min=0"`
```

Comment:
@soumya-codes - did you mean uint16?

Reply:
Agree on using `MaxClients`. Using this with int32 type instead of `MaxCmdHandlers`. I

 agree we can change `MaxClients` type to `uint16`. Please confirm

Reply:
Finally I used uint32 because this value is compared with actual number which is atomic.Uint type and atomic support Uint32 and Uint64 not Uint16.